src/game.cpp

```diff
@@ -5630,6 +5631,17 @@ bool game::npc_menu( npc &who )
             add_msg( _( "You cannot swap places while grabbing something." ) );
         }
     } else if( choice == push ) {
+        if( !obeys ) {
+            if( !query_yn( _( who.name + " may be upset by this. Continue?" ) ) ) {
```

Comment:
Maybe `!query_yn( _( "%s may be upset by this. Continue?" ), who.name )` to make it easier for translators?

Reply:
How would it make it easier for translators?

Reply:
Alright, I just learned why. Change has been made.